modules/swagger-annotations/src/main/java/io/swagger/oas/annotations/callbacks/Callback.java

```diff
@@ -39,18 +35,21 @@
 @Inherited
 public @interface Callback {
   /**
-   * the friendly name used to refer to this callback
+   * The friendly name used to refer to this callback
+   * @return the name of the callback
    **/
   String name() default "";
 
-  /**
-   * an absolute URL which defines the destination which will be called with the supplied operation definition
-   **/
+/**
+ * An absolute URL which defines the destination which will be called with the supplied operation definition.  
+ * @return the callback URL
+ */
   String callbackUrlExpression() default "";
 
   /**
-   * the operation that will be executed on callback
+   * The array of operations that will be called out-of band 
+   * @return the callback operations
    **/
-  Operation[] operation() default @Operation();
+  Operation[] operation() default {};
```

Comment:
See issue https://github.com/swagger-api/swagger-core/issues/2272.    I am curious as to why we would want the default of an array of operations to be a single empty operation?  Shouldn't be an empty array?